etna/transforms/outliers.py

```diff
@@ -41,11 +63,13 @@ def fit(self, df: pd.DataFrame) -> "OutliersTransform":
 
         Returns
         -------
-        result: _OneSegmentTimeSeriesImputerTransform
+        result: OutliersTransform
```

Comment:
You need to remove `OutliersTransform` as it is already defined at the line 
`def _save_original_values(self, ts: TSDataset) -> "OutliersTransform":`